addons/packages/antrea/1.5.2/bundle/config/overlay/antrea_overlay.yaml

```diff
@@ -0,0 +1,421 @@
+  AntreaIPAM: #@ values.antrea.config.featureGates.AntreaIPAM
+
+  #! Enable managing external IPs of Services of LoadBalancer type.
+  ServiceExternalIP: #@ values.antrea.config.featureGates.ServiceExternalIP
+
+#! The port for the antrea-controller APIServer to serve on.
+#! Note that if it's set to another value, the `containerPort` of the `api` port of the
+#! `antrea-controller` container must be set to the same value.
+#!apiPort: 10349
+
+#! Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
+#!enablePrometheusMetrics: true
+
+#! Indicates whether to use auto-generated self-signed TLS certificate.
+#! If false, A Secret named "antrea-controller-tls" must be provided with the following keys:
+#!   ca.crt: <CA certificate>
+#!   tls.crt: <TLS certificate>
+#!   tls.key: <TLS private key>
+#! And the Secret must be mounted to directory "/var/run/antrea/antrea-controller-tls" of the
+#! antrea-controller container.
+#! selfSignedCert: true
+
+#! Comma-separated list of Cipher Suites. If omitted, the default Go Cipher Suites will be used.
+#! https://golang.org/pkg/crypto/tls/#pkg-constants
+#! Note that TLS1.3 Cipher Suites cannot be added to the list. But the apiserver will always
+#! prefer TLS1.3 Cipher Suites whenever possible.
+tlsCipherSuites: #@ values.antrea.config.tlsCipherSuites
+
+#! TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
+#!tlsMinVersion:
+
+#! If Antrea is upgraded from version <= v0.13 and legacy CRDs are used, this option should be
+#! enabled, otherwise the CRDs created with the legacy API groups will not take any effect and
+#! work as expected. When the mirroring is enabled, if a legacy CRD is created with legacy API
+#! groups, mirroring-controller will create a new CRD with the Spec and Labels from the legacy
+#! CRD. Afterwards, the modification of Spec and Label in legacy CRD will be synchronized to new
+#! CRD automatically. In addition, the modification of Status in new CRD will also be synchronized
+#! to legacy CRD automatically. If a legacy CRD is deleted, the corresponding new CRD will be deleted.
+#! Note that: to decouple a new CRD from the corresponding legacy CRD, the legacy CRD should be
+#! annotated with "crd.antrea.io/stop-mirror". Afterwards, updates to the legacy CRDs will no
+#! longer be reflected in the new CRD, and all CRUD operations should be done through the new
+#! API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
+#! new CRDs.
+#! legacyCRDMirroring: true
+
+nodeIPAM:
+#! Enable the integrated Node IPAM controller within the Antrea controller.
+#!  enableNodeIPAM: #@ values.antrea.config.nodeIPAM
+
+#! CIDR ranges for Pods in cluster. String array containing single CIDR range, or multiple ranges.
+#! The CIDRs could be either IPv4 or IPv6. At most one CIDR may be specified for each IP family.
+#! Value ignored when enableNodeIPAM is false.
+#!  clusterCIDRs: []
```

Comment:
In TKG, the node IPAM controller should always be enabled in kube-controller-manager by default. I think usually it's not necessary to enable Antrea nodeIPAM. Antrea nodeIPAM controller does the same work as node IPAM controller in kube-controller-manager. Maybe we can either:

- Remove nodeIPAM related feature gate and configs from values.yaml, and keep it disabled.

or

- Keep nodeIPAM variables in values.yaml, and also expose nodeIPAM.clusterCIDRs in values.yaml. This is for Antrea nodeIPAM to work, clusterCIDRs must be specified.